com.unity.multiplayer.mlapi/Runtime/Messaging/InternalMessageHandler.cs

```diff
@@ -139,7 +137,8 @@ public void HandleAddObject(ulong clientId, Stream stream)
         {
             using (var reader = PooledNetworkReader.Get(stream))
             {
-                var isPlayerObject = reader.ReadBool();
+                var isPlayerObjectI = reader.ReadByte();
+                bool isPlayerObject = isPlayerObjectI != 0;
```

Comment:
o.o huh?

Reply:
Ahahaha... I'm glad you noticed this. I did this when I was trying to debug why what I was receiving here wasn't what I was sending and the bool was truncating all non-zero values to `true` so I did that so I could see the actual value. I meant to revert it.